jaxrs/src/main/java/se/fortnox/reactivewizard/jaxrs/SuppressPathCollision.java

```diff
@@ -0,0 +1,11 @@
+package se.fortnox.reactivewizard.jaxrs;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+@Retention(RetentionPolicy.RUNTIME)
+@Target(ElementType.METHOD)
+public @interface SuppressPathCollision {
```

Comment:
There is already a java.lang.SuppressWarnings annotation that could be used I guess. Then the users of the annotation would not have to add jaxrs module to classpath (it is not part of dependencies in api-modules). If you for some reason need to create a new annotation, then perhaps it is nice to have it in module `jaxrs-api`? Or perhaps this should just go on the impl-class and then jaxrs-module will be on CP anyway...

Reply:
The SuppressWarnings does not have runtime retention so I had to create my own annotation.
Moving it to the jaxrs-api mdule sounds good though.